tempto-examples/src/main/resources/tempto-configuration.yaml

```diff
@@ -84,3 +84,9 @@ list:
     - ala
     - ma
     - kota
+
+ldap:
+  url: ldap://ldapserverhost
```

Comment:
Now when ldap support is in separate jar (`tempto-ldap`) add it as dependency of tempto-examples so this part of config is used.